<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructEdDlg</class>
 <widget class="QWidget" name="StructEdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vStructureEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pencilButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VoxCad.qrc">
           <normaloff>:/new/Icons/Icons/Pencil.png</normaloff>:/new/Icons/Icons/Pencil.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="squareButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VoxCad.qrc">
           <normaloff>:/new/Icons/Icons/Rectangle.png</normaloff>:/new/Icons/Icons/Rectangle.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="ellipseButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VoxCad.qrc">
           <normaloff>:/new/Icons/Icons/Ellipse.png</normaloff>:/new/Icons/Icons/Ellipse.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="layerbackButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VoxCad.qrc">
           <normaloff>:/new/Icons/Icons/LayerBack.png</normaloff>:/new/Icons/Icons/LayerBack.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="layerforwardButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VoxCad.qrc">
           <normaloff>:/new/Icons/Icons/LayerForward.png</normaloff>:/new/Icons/Icons/LayerForward.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="refviewButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VoxCad.qrc">
           <normaloff>:/new/Icons/Icons/Aux_View.png</normaloff>:/new/Icons/Icons/Aux_View.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="MatLabel">
       <property name="text">
        <string>Material:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="MatList"/>
     </item>
     <item>
      <widget class="QPushButton" name="DoneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VoxCad.qrc"/>
 </resources>
 <connections/>
</ui>
